### ISSUE TITLE
PGNAPES exports

### ISSUE THREAD

**Issue by LeonHAV** [ices-eg/wg_WGAcousticGov#30]
Remember to implement the exports from acoustic and biotic in PGNAPES format.

**Comment by HjalteParner** [comment]
I was just looking at some old documents on the PGNAPES format and an attempt to mapping the PGNAPES format to the ICES acoustic and biotic format - please see attached. Even though the format used in the mapping document is outdated, the requirement for multiple mappings (some hard to do) stands.

In my view I don't think it can be justified to produce the PGNAPES (one of potential many formats/systems out there) format from the Acoustic Trawl data portal but I think it would be more appropriate and straight forward for PGNAPES to harvest the ICES Acoustic and Biotic data files and then inject these into the PGNAPES database as seen fit. The formats are well defined and the vocabularies public available. Harvesting can be done using already available web services automatically and/or manually through the interface.

Above seen together with the long term goal that TAF will take over the calculation/reporting at some point which currently seems to be the main reason for keeping PGNAPES alive for some survey group presumably being dependant on accompanying data (uncalibrated hydrography and zooplankton) and/or historical data within the PGNAPES database.

[Final database worksheets_2.docx](https://github.com/ices-eg/wg_WGACOUSTICGOV/files/8002835/Final.database.worksheets_2.docx)

[Mapping PGNAPES to ICES Acoustic database.docx](https://github.com/ices-eg/wg_WGACOUSTICGOV/files/8002838/Mapping.PGNAPES.to.ICES.Acoustic.database.docx)

**Comment by JanArge** [comment]
According to Leon, the suggestion was that you (promised to) make a small "harvest" routine such that PGNAPES can harvest the data from the (complicated) acoustic trawl portal -once we managed to export our surveys into the ICES database, which we did last year. 
The justification to keep PGNAPES alive is that it is the only reporitory that holds the complete historic data series from the coordinated surveys in the Nordic Seas (IBWSS, IESNS and IESSNS) back to around 1995 in one simple repository. This inclues not only biotic and acoustic data, but also zooplankton data along with hydrograpic data.
